README.rst

```diff
@@ -27,11 +27,11 @@ Dependencies
 
 Requirements:
 
-- ``ansible`` (`>= 2.9.5`), ``yq``, ``jq``
+- ``ansible`` (`>= 2.9.5`), ``yq``, ``jq``, ``fire``
```

Comment:
Good catch.

Maybe let's just point users to the `requirements.txt` file, rather than maintaining these too lists